eisbw/src/main/java/eisbw/actions/SendText.java

```diff
@@ -0,0 +1,131 @@
+package eisbw.actions;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import eis.iilang.Action;
+import eis.iilang.Parameter;
+import eis.iilang.Identifier;
+import jnibwapi.JNIBWAPI;
+import jnibwapi.Unit;
+import jnibwapi.types.UnitType;
+import org.apache.commons.lang3.concurrent.TimedSemaphore;
+
+/**
+ * @author Thijs Molendijk & Thijs Raymakers - Send a chat message to all players in the game.
+ */
+public class SendText extends StarcraftAction {
+	private static List<String> CHEATS = Arrays.asList(
+			"show me the money",
+			"whats mine is mine",
+			"breathe deep",
+			"something for nothing",
+			"operation cwal",
+			"modify the phase variance",
+			"medieval man",
+			"the gathering",
+			"noglues",
+			"power overwhelming",
+			"food for thought",
+			"there is no cow level",
+			"ophelia",
+			"staying alive",
+			"man over game",
+			"game over man",
+			"war aint waht it used to be",
```

Comment:
Did you make a typo here or is this the actual cheat code?

Reply:
This seems to be a typo.